cpp/core/compute/Runtime.h

```diff
@@ -164,7 +139,12 @@ class ExecutionCtx : public std::enable_shared_from_this<ExecutionCtx> {
     return taskInfo_;
   }
 
+  const std::unique_ptr<ObjectStore>& objectStore() {
```

Comment:
Runtime keep unique ref of ObjectStore, I think return raw pointer would be more make sense which mean caller should not hold it.